modular_zubbers/code/modules/rr_opt_in/code/objective.dm

```diff
@@ -0,0 +1,69 @@
+/datum/objective
+	/// The default opt in level of this objective. Only targets with opt in above or at this will be considered for this objective.
+	var/default_opt_in_level = RR_OPT_OUT
+
```

Comment:
Ditto, we could rename this to convey the thing and just use TRUE/FALSE
```suggestion
	/// The default opt in level of this objective. Only targets with opt in above or at this will be considered for this objective.
	var/needs_rr_enabled = FALSE

```